storybook/.storybook/preview.ts

```diff
@@ -41,6 +42,7 @@ const preview: Preview = {
     backgrounds: {
       disable: true,
     },
+    internalWrapper: false,
```

Comment:
Prefer something like?

```suggestion
    storyContext: 'MODULE_UI' as 'MODULE_UI' | 'STANDALONE',
```